source/dm_recipes.rst

```diff
@@ -0,0 +1,56 @@
+=======================
+Data Management Recipes
+=======================
+
+----------------
+Register context
+----------------
+
+Data Provider registers dataset
+===============================
+
+In order to make a dataset findable, a dataset must be registered in S-ENDA Find with appropriate metadata, which is indexed and exposed in CSW. The procedure is as follows:
+
+#. Create MMD XML record of metadata
+#. Translate from MMD to ISO19115
+#. Index in CSW
+#. Expose ISO19115 metadata via CSW to the internet
+
+This is done with the s-enda-find Virtual Machine (VM), for which a Vagrant configuration is available in https://github.com/metno/S-ENDA-Prototype.
+
+**TODO:** We need to describe how to do this:
+
+* manually in the VM
+* through the geoassets API
+
+S-ENDA Find gives feedback
```

Comment:
Today the feedback is just a mock, but maybe you mean to describe how we think this could be done?

Reply:
exactly :+1: